bitnami/wordpress/values.yaml

```diff
@@ -694,3 +694,38 @@ externalDatabase:
 ## NOTE: Combined with extraVolume and extraVolumeMounts to mount the configmap to /docker-entrypoint-init.d where custom user init scripts are looked for
 ##
 customPostInitScripts: {}
+
+##
+## Init containers parameters:
+## volumePermissions: Change the owner of the persist volume mountpoint to RunAsUser:fsGroup
+##
+volumePermissions:
+  enabled: false
+  image:
+    registry: docker.io
+    repository: bitnami/minideb
+    tag: buster
+    pullPolicy: Always
+    ## Optionally specify an array of imagePullSecrets.
+    ## Secrets must be manually created in the namespace.
+    ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
+    ##
+    # pullSecrets:
+    #   - myRegistryKeySecretName
+  resources: {}
+  # resources:
+  #   requests:
+  #     memory: 128Mi
+  #     cpu: 100m
```

Comment:
You added the parameters `volumePermissions.resources.limits/requests` to the README but forgot to include them here. You can use [bitnami/cassandra](https://github.com/bitnami/charts/blob/6e457f514abf626f8eb57d4fdf5f681384623f24/bitnami/cassandra/values.yaml#L134) as reference.